constants/links.js

```diff
@@ -0,0 +1,95 @@
+const QISKIT_URL = 'https://qiskit.org'
```

Comment:
Is there some reason for mixing _upper_snake_case_ with _camelCase_ for `const` names? If not, would you mind to choose one and stick with it, please?